addons/search-sprites/addon.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "Search sprites by name",
+  "name": "Sprite search",
```

Comment:
This sounds more like a sharable sprite library. I prefer the old name over anything else I can think of

Reply:
How about "search sprites in project"?

Reply:
`Project sprite search`?

Reply:
I like that.
```suggestion
  "name": "Search box in sprite pane",
```